docs/release_notes/2024/june2024.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 1
+id: June_2024
+description: Release notes for June
+title: June 2024
+tags:
+  - release notes
+  - changelog
+  - june
+---
+
+## 3.3.7.\* (June 7,2024)
+
+- Prophecy Python libs version: 1.9.4
+- Prophecy Scala libs version: 8.0.5
+
+### Features {#Features337}
+
+#### Copilot AI improvements
+
+- **Auto documentation and description**: The AI experience in Spark and SQL copilot is now automated. This includes automatic documentation and descriptions for datasets, columns, projects, and more. Additionally, Git commit and release messages are now auto-generated.
+
+- **Conversational interface**: Users can now interactively modify existing models after generating them with the Copilot, allowing for continuous, incremental prompts through a chat like interface.
+
+- **UDFs and Macros in copilot expression predictions**: The copilot can now predict expressions using custom UDFs and Macros defined in Spark and SQL projects.
+
+- **Data Context in Prompts**: Users can enable data context for expression prediction, ensuring the use of customer data context in Gems. This feature can be enabled by a Team admin in the Team level settings.
+
+Along with above, Various UX improvements have been made to ensure a seamless AI experience in our Data Transformation Copilots.
+
+#### Spark Copilot enhancements
+
+- **Business Rule Engine**: Added support for Business Rules in Spark Copilot. Users can define business rules as functions and use them in any of their transformation Gems to apply.
+  Please note User defined functions are also moved to the left side project Browser with this change.
+
+#### SQL Copilot Enhancements
+
+- **Auto AI suggestions in SQL code editor**: The SQL copilot now features AI auto-suggestions and fixes in the SQL **code view** too, facilitating easier model modification directly in code.
```

Comment:
`The SQL copilot now features AI auto-suggestions in the SQL`

we don't have code fixes in the editor yet.